backend/demo_phase1.sh

```
@@ -0,0 +1,62 @@
+#!/bin/bash
+
+# Phase 1 Architecture Demo
+# This script demonstrates the new clean architecture setup
+
+echo "🏗️  Phase 1 Architecture Demo - Poo Tracker Backend Refactor"
+echo "=============================================================="
+echo
+
+echo "📁 Directory Structure:"
+echo "------------------------"
+cd /home/kjanat/Projects/poo-tracker/backend
```

Shellcheck (0.10.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)